templatestore/frontend/src/components/Dropdowncomp.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from '../style/templateScreen.less';
+export default function Dropdowncomp(props) {
+    return (
+        <div className={styles.teAttributesRow}>
+            <div className={styles.teAttributesCell}>
+                <select
+                    name="urlKey"
+                    value={props.urlKey}
+                    placeholder="Choose a value"
+                    onChange={(e) => props.handleChange(e, props.id)}
+                >
+                    <option value="" disabled>Choose URL</option>
+                    {props.listOfUrls.map((urlopt) => (
```

Comment:
`urlList` would be better instead of  `listOfUrls`, and
`url` would be better instead of `urlopt`.
Can you think of even better alternatives?

Reply:
I think urlKeyList and urlKey would be better since they don't really store the url, but their key.